pkg/runtime/runtime.go

```diff
@@ -3150,14 +3297,46 @@ func ShutdownSignal() chan os.Signal {
 	return stop
 }
 
+func (a *DaprRuntime) createHTTPEndpointsChannels() (map[string]channel.HTTPEndpointAppChannel, error) {
+	// Create dedicated app channels for known app endpoints
+	endpoints := a.compStore.ListHTTPEndpoints()
+
+	if len(endpoints) > 0 {
+		channels := make(map[string]channel.HTTPEndpointAppChannel, len(endpoints))
+
+		for _, e := range endpoints {
+			pipeline, err := a.buildAppHTTPPipeline()
```

Comment:
Please move this outside of the loop so it's the same object for all endpoints. Since it's not endpoint-dependent (at least not today), there's no need to allocate multiple instances of this

Reply:
Moved